.mergify.yml

```
@@ -51,7 +51,14 @@ pull_request_rules:
       label:
         add:
           - new CVE
+
 merge_protections:
+  - name: No Manual Merge
+    if:
+      - label = manual merge
+    success_conditions:
+      - label != manual merge
```

This should also work, right ?
```suggestion
    if: []
    success_conditions:
      - label != manual merge
```

---

This will just always display the rule, instead of just when the label is added